spec/unit/order_clause_spec.rb

```diff
@@ -67,6 +67,26 @@
     end
   end
 
+  describe "hstore_col#>>'field'_asc" do
+    let(:clause) { "hstore_col->>'field'_asc" }
+
+    it { is_expected.to be_valid }
+
+    describe '#field' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.